app/router.js

```diff
@@ -8,6 +8,8 @@ const Router = EmberRouter.extend({
 
 Router.map(function() {
   this.route('users');
+  this.route('teams');
+  this.route('teamss');
```

Comment:
Kindly use self explaining names for variables, properties and routes rather than using like `a`, `teams1` and `teamss` etc.